pallets/offchain-demo/Cargo.toml

```
@@ -30,5 +31,6 @@ std = [
 	'frame-system/std',
 	'sp-io/std',
 	'sp-runtime/std',
-	'sp-std/std'
+	'sp-std/std',
+	'simple-json2/std'
```

```suggestion
	'simple-json2/std',
```
This trailing comma makes diffs cleaner when you add stuff later. See how this change also affected the `sp-std` line even though that line didn't change in any meaningful way?